features/support/fs.rs

```diff
@@ -97,6 +101,10 @@ pub fn create_project() -> io::Result<Project> {
 }
 
 fn create_cargo_toml(dir: TempDir) -> io::Result<TempDir> {
+  fn double_backslash<T: Display>(path: T) -> String {
```

Comment:
Oh I'm dumb. You're calling .display() on current_dir().unwrap(), so of course all you have is Display. LGTM